tests/integrations/config/test-custom.toml

```diff
@@ -3,6 +3,14 @@ log-file = "foo"
 log-rotation-timespan = "1d"
 panic-when-unexpected-key-or-data = true
 
+[readpool]
+use-yatp = true
```

Comment:
Maybe we need another name for it.. "Yatp" is the internal implementation name but user won't understand what it is (and it is an acronym as well, even harder to understand. Its full name "yet another thread pool" is still hard to understand..). How about simply call it `use-unified-thread-pool`?